kv/store_base.go

```diff
@@ -495,6 +501,18 @@ func indexByOrgNameKey(orgID influxdb.ID, name string, caseSensitive bool) ([]by
 	}
 	k := make([]byte, influxdb.IDLength+len(name))
 	copy(k, orgIDEncoded)
+
+	if name == "" {
```

Comment:
I would do it before make k

Reply:
@kelwang, the comment below here explains why this is here. It is to provide a partial key when warranted. Fairly often we lookup entities by `orgID` without a name, this provides that mechanism. When no name is present, it still reponds with an error, but also responds with the key filled with the `orgID`